enterprise/replay/replay.go

```diff
@@ -152,6 +154,7 @@ func (handle *Handler) Setup(ctx context.Context, dumpsLoader *dumpsLoaderHandle
 	handle.noOfWorkers = config.GetEnvAsInt("WORKERS_PER_SOURCE", 4)
 	handle.dumpsLoader = dumpsLoader
 	handle.dbReadSize = config.GetEnvAsInt("DB_READ_SIZE", 10)
+	handle.dsQueryLimit = config.GetEnvAsInt("DS_QUERY_LIMIT", 10)
```

Comment:
imposing a limit is not something we would like to do normally, thus let's start with no limit: 0